myapp/scheduler/tests/test_stock_api_update.py

```diff
@@ -0,0 +1,14 @@
+from unittest import TestCase
+
+from ..stock_api_update import stock_api_update
+from ...models import Stock
+
+
+class JbsTestCase(TestCase):
+
+    def test_api_update(self):
+        stock_api_update()
+        value_1 = Stock.objects.filter(top_rank=1)[0].last_modified
+        stock_api_update()
+        value_2 = Stock.objects.filter(top_rank=1)[0].last_modified
+        self.assertNotEqual(value_1, value_2)
```

Comment:
Did not Malak write a similar test some time ago (?)

Reply:
no